doc/ideas/fr_technical_overview.md

```diff
@@ -0,0 +1,137 @@
+Gameplay           | GM19       | Scale wonder cost to max pop, number of players and available resources.
+Gameplay           | GM20       | a new 5th "free trade" age where allies share resources and give each other massive battles.
+Gameplay           | GM21       | New unit: Lance Cavalry, run over units in defending formations
+Gameplay           | GM22       | Researches can be globally and locally applied. e.g. upgrade just units garrisoned in the barracks for less research cost; e.g. you need 3 pikemen for the battle now, but you don't want to produce them in the future in masses (see GM18)
+Gameplay           | GM23       | Units can level up by gaining experience doing what they normally do, fighting in battle against their conter unit will give more XP against that unit style. Villagers chopping a forest nothing map might be a real pro at chopping trees in the end of the game.
+Gameplay           | GM24       | Queue for technology researches.
+Gameplay           | GM25       | Farms cannot be placed on desert ground.
+Gameplay           | GM26       | Improved water battles.
+Gameplay           | GM27       | Percentage based armor system similar to AoEO and AoM. Will need to rebalance whole gameplay.
+Gameplay           | GM28       | empty
+Gameplay           | GM29       | Entirely rebalance milita line. Main gold unit similar to knights/archers. Instead of a counter unit currently.
+Gameplay           | GM30       | Nerf the range on towers early and make them scale much better into the late game.
+Gameplay           | GM31       | Unit queuing doesn't reserve resources. Resources are taken in account, when the unit is actually starting to produce.
+Gameplay           | GM32       | Make denying upgrades more easy, like SC2. often players will destroy buildings to cancel an upgrade in progress and gain the upgrade advantage. maybe in a special mod with less hp for buildings. everybody is free to design such a mod.
+Gameplay           | GM33       | Possibility to put archers/ranged units on a wall/castle to get elevation bonus.
+Gameplay           | GM34       | Quickwalling: first XX% (variable percentage) of a foundation is walkable, setting for e.g. tournaments
+Gameplay           | GM35       | Mesoamerican civilizations (already the most novel civs in the game) may have a unique tech tree. Possibility for a unique tech tree in API.
+Gameplay           | GM36       | Controlled Forest-Burns: Allows the player (technology Researched at the University in the Castle age) to destroy Forests to create more space; this goes much faster/efficient than Chopping, but obviously at the expense of the Lumber and generates Regular Terrain.
+Gameplay           | GM37       | Give siege Rams an upgrade that prevents unit production while it's attacking a building. (5-second debuff that refreshes each attack). Note for me: Check if current API is ready to change that easily?
```

Comment:
Rams --> rams

Remove "Note for me" line

Reply:
done